webui/src/Views/InstanceView.elm

```diff
@@ -363,30 +363,37 @@ getActiveAllocations task =
 
 instanceTasksView : Maybe (List Task) -> Html msg
 instanceTasksView instanceTasks =
-    case Maybe.map (List.concatMap getActiveAllocations) instanceTasks of
-        Nothing ->
-            h5 []
-                [ text "Loading allocated tasks "
-                , i [ class "fa fa-spinner fa-spin" ] []
-                ]
-
-        Just [] ->
-            h5 [] [ text "No allocated tasks" ]
-
-        Just allocations ->
-            div []
-                [ h5 [] [ text "Allocated tasks" ]
-                , table [ class "table table-condensed table-hover" ]
-                    [ thead []
-                        [ tr []
-                            [ th [] [ text "Task" ]
-                            , th [] [ text "Allocation ID" ]
-                            , th [] [ text "State" ]
+    let
+        allocatedTasks =
+            case Maybe.map (List.concatMap getActiveAllocations) instanceTasks of
+                Nothing ->
+                    [ i [ class "fa fa-spinner fa-spin" ] [] ]
+
+                Just [] ->
+                    [ text "No tasks have been allocated, yet." ]
+
+                Just allocations ->
+                    [ table [ class "table table-condensed table-hover" ]
+                        [ thead []
+                            [ tr []
+                                [ th [] [ text "Task" ]
+                                , th [] [ text "Allocation ID" ]
+                                , th [] [ text "State" ]
+                                ]
                             ]
+                        , tbody [] <| List.indexedMap instanceAllocationRow allocations
                         ]
-                    , tbody [] <| List.indexedMap instanceAllocationRow allocations
                     ]
+    in
+        div
+            [ style
+                [ ( "margin-bottom", "15px" ) ]
```

Comment:
Furthermore please add the margin to the top-level container of the expanded instance, not to an arbitrary div in between, so that we can safely add new elements to the expander without breaking the layout.